packages/katello/katello/katello-service

```diff
@@ -74,14 +73,7 @@ def manage_services(action)
 
   services_by_priority(action == 'stop').each do |service|
     if service_exists?(service)
-      case action
-      when 'enable', 'disable'
-        puts "systemctl #{action} #{service}.service"
-        puts `systemctl #{action} #{service}.service`
-      else
-        puts "#{COMMAND} #{service} #{action}"
-        puts `#{COMMAND} #{service} #{action}`
-      end
+      puts `#{COMMAND} #{action} #{service}`
```

Comment:
This no longer prints what it does. I think we should stay compatible. It also no longer appends `.service`. Technically not needed but I'd keep it in since it's more explicit.